build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-turing/build.sh

```diff
@@ -45,4 +45,5 @@ echo "Building with cmake"
 ./build_tools/cmake/clean_build.sh
 
 echo "Testing with ctest"
+export IREE_CUDA_DISABLE=0
```

Comment:
Does this need to happen in this pull request? Other changes are for creating docker images. This is actually for testing using those docker images, which can happen later when we are clear the docker images work.

Reply:
We can test the docker images work by adding this in this PR. Otherwise we're changing the docker images but don't know if they actually enable cuda, which feels a bit pointless

Reply:
Oh okay you are using this pull request for testing. I was thinking this pull request is just for landing the changes for creating docker images. Ignore this then. :)